src/main/java/com/solr/clientwrapper/domain/service/SolrSchemaStorageService.java

```diff
@@ -0,0 +1,73 @@
+package com.solr.clientwrapper.domain.service;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.apache.solr.client.solrj.request.schema.FieldTypeDefinition;
+import org.apache.solr.client.solrj.request.schema.SchemaRequest;
+import org.apache.solr.client.solrj.response.schema.SchemaResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.port.api.SolrSchemaStorageServicePort;
+
+@Service
+@Transactional
+public class SolrSchemaStorageService implements SolrSchemaStorageServicePort {
+	
+	private static String DEFAULT_COLLECTION = "techproducts";
+	
+	String urlString = "http://localhost:8985/solr/";
+	
+	SolrClient solr = new HttpSolrClient.Builder(urlString+DEFAULT_COLLECTION).build();
+	
+	@Override
+	public String validateSchema() {
+		System.out.println("!!!!!!!!!!11 %%%%%%%%%%%%%%%% 11!!!!!!!!!!!!!!");
```

Comment:
In the entire code don't use System.out.println rather use loggers

Reply:
Yes, that method was just to tests some stuffs. Had to remove at last, will do that.